installer/data/mysql/sysprefs.sql

```diff
@@ -30,6 +30,7 @@ INSERT INTO systempreferences ( `variable`, `value`, `options`, `explanation`, `
 ('AllowNotForLoanOverride','0','','If ON, Koha will allow the librarian to loan a not for loan item.','YesNo'),
 ('AllowOfflineCirculation','0','','If on, enables HTML5 offline circulation functionality.','YesNo'),
 ('AllowPatronToSetCheckoutsVisibilityForGuarantor',  '0', NULL, 'If enabled, the patron can set checkouts to be visible to his or her guarantor',  'YesNo'),
+('AllowPatronsManageAPIKeysInOPAC', '0', NULL, 'If enabled, patrons can manage their own API keys in the OPAC',  'YesNo'),
```

Comment:
A 'Client Credentials Grant' should really never be used in this way... it's all about authenticating a client and has nothing to do with authenticating a client on behalf of a user.

Reply:
You propose we strip that part out?

Reply:
Yes, it should be added back in once proper thought has gone into A) Whether we wish to support client credentials flow and thus trust users to understand the security issues involved or B) Implement code grant flow fully.

Reply:
In either case, we really should be adding a step to allow the user to select which of their permissions they wish to allow the remote client to assume.